pkg/cmd/pulumi/import.go

```diff
@@ -531,6 +531,9 @@ func newImportCmd() *cobra.Command {
 			}
 
 			if res != nil {
+				// TODO: This helpful message prints even in the case of the user doing a successful preview and then
+				// choosing "no" to not actually do the import. We need a way to distinguish Import _failing_ vs
+				// being canceled by user request.
 				fmt.Print("Import failed, try specifying the set of properties to import with.\n")
```

Comment:
I might be missing this check, but are we confirming that the set of properties is not provided before we display this message.

Reply:
Good point. What happens if a specify a "wrong" set of properties?

Reply:
Oh good point! Maybe something as simple as:
`fmt.Print("Import failed, try specifying a different set of properties to import with.\n") `

Reply:
I guess it could be useful to differentiate our hint in case of no user-provided properties vs. with a wrong set of properties?